x-pack/platform/plugins/shared/fleet/public/applications/integrations/sections/epm/screens/installed_integrations/types.tsx

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { PackageListItem } from '../../../../../../../common';
+
+export type PackageInstallationStatus =
+  | 'not_installed'
+  | 'installing'
+  | 'installed'
+  | 'install_failed'
+  | 'upgrade_failed'
+  | 'upgrade_available';
+
+export type PackageListItemWithExtra = PackageListItem & {
+  extra: {
+    installation_status: PackageInstallationStatus;
```

Comment:
I was confused a little with the naming here since it's very similar to `EpmPackageInstallStatus`

can we consider a different name that makes it more clear that this is only for the UI table state? something like `InstalledPackageUICondition` (naming is hard...)

Reply:
what do you think of something like moving from an `extra` property to `ui` and use a more explicit type
```
PackageListItem & {
  ui: {
    installation_status: InstalledPackageUICondition;
}
```

Reply:
Done in https://github.com/elastic/kibana/pull/212582/commits/81a06e7913f74042c3b5ad88fb0a7bae22af3e52